chroot-bin/croutonwheel

```diff
@@ -12,11 +12,27 @@ l=6
 r=7
 # Do horizontal scrolling by default
 x=1
-# Set the default scaling and constant values
-xs='0.2'
-xc='0.01'
-ys='0.2'
-yc='0.05'
+
+if [ -L /etc/crouton/cmt ]; then
+  # Enable xinput Australian Scrolling
+  xinput --set-prop --type=int --format=8 "Atmel maXTouch Touchpad" "Australian Scrolling" 1
```

Comment:
We're either going to need to provide a way to configure this, or detect it from the DE's settings if possible.

Also, we need to figure out if it's the same on Cypress and other trackpads...